webpack/assets/javascripts/foreman_tools.js

```diff
@@ -122,8 +122,8 @@ export function updateTable(element) {
     .text()
     .trim();
   uri.setSearch(values);
-  /* eslint-disable no-undef */
-  Turbolinks.visit(uri.toString());
+
+  window.location.href = uri.toString();
```

Comment:
Since 184f8d5ce23cfe208a696535bcea346074d8600a the entire `updateTable` function is unused in Foreman itself. Don't know about plugins but I'd appreciate it if someone would at least deprecate it but possibly just remove it.

Reply:
I've deprecated it, just in case.